src/Repositories/ChildRepository.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Choredo\Repositories;
+
+use Choredo\Filter;
+use Choredo\LimitOffset;
+use Choredo\Sort;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\Tools\Pagination\Paginator;
+
+class ChildRepository extends EntityRepository
+{
+    /**
+     * @param LimitOffset $pagination
+     * @param Sort[] $sorts
+     * @param Filter[] $filters
+     * @return Paginator
+     */
+    public function getAll(LimitOffset $pagination, array $sorts = [], array $filters = []): Paginator
```

Comment:
This needs to filter by family, no?

Reply:
I added the filter in this PR but it needs to be enforced at the EM level, I'll activate it in the endpoints PR.